sample.cpp

```diff
@@ -142,6 +149,9 @@ int main(int argc, char** argv)
 
 	world->tick(10.f);
 
+	ent->remove<Position>();
+	ent->remove<Rotation>();
```

Comment:
The two `ent->remove<>()` lines need to be called after the `cout` below, otherwise you're accessing deleted objects. The cout line should display the position as `(10, 10)` and rotation as `20`.

Reply:
done